src/modules/service-catalog/ServiceCatalogList.tsx

```diff
@@ -101,6 +106,17 @@ export function ServiceCatalogList({
 
   return (
     <Container>
+      <span>
+        {t("service-catalog.service-count", {
+          "defaultValue.zero": "{{count}} services",
+          "defaultValue.one": "{{count}} service",
+          "defaultValue.two": "{{count}} services",
+          "defaultValue.few": "{{count}} services",
+          "defaultValue.many": "{{count}} services",
+          "defaultValue.other": "{{count}} services",
```

Comment:
There is no need to specify all the possible variants here since these are default values used only for English if the string is not present in the translation files. We can just specify the `one` and `other` variants, which are the only ones needed for English